<launch>
  
  <node
      name="ros_traj_consum_interpol_interface"
      pkg="ros_pybullet_interface"
      type="ros_traj_consum_interpol_interface_node.py"
      output="screen">

    <!-- Traj config (required) -->
    <param
        name="traj_config"
        value="{ros_pybullet_interface}/configs/traj_object3D_freemotion.yaml"
        type="str"/>

  </node>

  <node
      name="ros_traj_consum_interpol_interface_sphere1"
      pkg="ros_pybullet_interface"
      type="ros_traj_consum_interpol_interface_node.py"
      output="screen">

    <!-- Traj config (required) -->
    <param
        name="traj_config"
        value="{ros_pybullet_interface}/configs/traj_sphere1.yaml"
        type="str"/>

  </node>

  <node
      name="ros_traj_consum_interpol_interface_sphere2"
      pkg="ros_pybullet_interface"
      type="ros_traj_consum_interpol_interface_node.py"
      output="screen">

    <!-- Traj config (required) -->
    <param
        name="traj_config"
        value="{ros_pybullet_interface}/configs/traj_sphere2.yaml"
        type="str"/>

  </node>

  <node
      name="ros_traj_consum_interpol_interface_Yang"
      pkg="ros_pybullet_interface"
      type="ros_traj_consum_interpol_interface_node.py"
      output="screen">

    <!-- Traj config (required) -->
    <param
        name="traj_config"
        value="{ros_pybullet_interface}/configs/traj_yang.yaml"
        type="str"/>

    <param
      name="robot_name"
      value="yang"
      type="str"/>

  </node>

  <node
      name="ros_traj_consum_interpol_interface_Yin"
      pkg="ros_pybullet_interface"
      type="ros_traj_consum_interpol_interface_node.py"
      output="screen">

    <!-- Traj config (required) -->
    <param
        name="traj_config"
        value="{ros_pybullet_interface}/configs/traj_yin.yaml"
        type="str"/>

    <param
      name="robot_name"
      value="yin"
      type="str"/>

  </node>


  <!-- Testing node trajectory publisher -->
  <node name="test_dual_arm_capturing_obj3D"
      pkg="ros_pybullet_interface"
      type="test_dual_arm_capturing_obj3D.py"
      output="screen">

<!--      <param-->
<!--          name="rotation_repr"-->
<!--          value="Quat"-->
<!--          type="str"/>-->

  </node>


</launch>
